File: Algorithms/ch3/remove_blanks.js
// Create a function that given a string, returns all of that string's contents, but without blanks.

function removeBlanks(str){
  var newString = ""
  for (var i = 0; i < str.length; i++){
    if (str[i] != " "){
      newString += str[i];
    }
  }
  return newString;
}

console.log(removeBlanks("I Love Coding!"));

File: Algorithms/ch4/remove_blanks.py
# Create a function that, given a string, returns all of that string's contents but without blanks.

#declare a function with a name

#test case for input that we assign to a variable

#parse through the string and find all the characters that are blanks

def remove_blanks(text):
    new_string = ""
    for char in text:
        if char != " ":
            new_string += char
    print new_string

remove_blanks("I love coding!")
